x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IndexLifecycle.java

```diff
@@ -320,15 +319,11 @@ private static List<NamedXContentRegistry.Entry> xContentEntries() {
                     new ParseField(SearchableSnapshotAction.NAME),
                     SearchableSnapshotAction::parse
                 ),
-                new NamedXContentRegistry.Entry(LifecycleAction.class, new ParseField(MigrateAction.NAME), MigrateAction::parse)
+                new NamedXContentRegistry.Entry(LifecycleAction.class, new ParseField(MigrateAction.NAME), MigrateAction::parse),
+                new NamedXContentRegistry.Entry(LifecycleAction.class, new ParseField(RollupILMAction.NAME), RollupILMAction::parse)
```

Comment:
Does this need to live behind the tsdb feature flag?

Reply:
Fixed in [86c3347](https://github.com/elastic/elasticsearch/pull/85708/commits/86c3347410aaf35140a79d839ff0494d1edcae35)